src/gui/pages_menu/KM_GUIMenuCampaign.pas

```diff
@@ -201,18 +201,14 @@ procedure TKMMenuCampaign.StartClick(Sender: TObject);
 
 procedure TKMMenuCampaign.AnimNodes(aTickCount: Cardinal);
 begin
-  if fCampaign <> nil then
+  if fCampaign = nil then Exit;
+  if Image_CampaignSubNode[fAnimNodeIndex].Visible then Exit;

```

Comment:
I checked. Everything is working.
Animation is on the nodes on the map, not until recently.
For example, we have 10 nodes on the map. Consequently cycle will go from 0 to 9 instead of from 0 to MAX_CAMP_NODES